packages/app/src/components/Event/EventList.js

```diff
@@ -36,39 +87,55 @@ const EventList = props => {
   );
 };
 
-const EventListQR = componentProps => {
-  return (
-    <QueryRenderer
-      environment={Environment}
-      query={graphql`
-        query EventListQuery {
-          events {
-            id
-            title
-            date
-            description
-            author
+const EventListPaginationContainer = createPaginationContainer(
```

Comment:
nice, but createRefetchContainer is easier

Reply:
Indeed, this is an older PR